src/vulnpy/trigger/__init__.py

```diff
@@ -12,6 +12,7 @@
     "hash": "hashme",
     "pt": "../../etc/passwd",
     "rand": "seed",
+    "sqli": "d', '4'),('e",
```

Comment:
How about using the classic attack str `"' or '1' = '1"`

Reply:
`' OR '1' =1'` is specific to the underlying db query, which is often a SELECT ... WHERE in our examples. To be able to reuse the same query as much as possible for all endpoints here, I used INSERT INTO, so `' OR '1' =1'` (or anything along those lines) would be a sql syntax error.

I could rework the queries to allow for this, or simply ignore the fact the attack would be an invalid query (it still hits the trigger, so it doesn't really matter). But I do think it would be nice to have some more diversity!